pkg/rehearse/jobs_test.go

```diff
@@ -248,16 +292,22 @@ func TestInlineCiopConfig(t *testing.T) {
 	}
 }
 
-func makeTestingPresubmit(name, context, branch string) *prowconfig.Presubmit {
+func makeTestingPresubmit(name, context, branch, variant string) *prowconfig.Presubmit {
```

Comment:
nit: I don't like helper functions like this in tests as they begin to accrue their own complex logic. Static test data is simpler and harder to mess up